media/css/style.css

```diff
@@ -53,25 +53,29 @@ GtkScrolledWindow, GtkViewport {
 
 .add_button {
     background-color: #91D064;
-    font: Bariol Bold 16;
+    font-family: "Bariol Bold";
+    font-size: 16pt;
     border-radius: 3px;
 }
 
-.add_button:insensitive {
+.add_button:disabled {
     background-color: #cdcdcd;
-    font: Bariol Bold 16;
+    font-family: "Bariol Bold";
+    font-size: 16pt;
     border-radius: 3px;
 }
 
 .cancel_button {
     background-color: #F37677;
-    font: Bariol Bold 16;
+    font-family: "Bariol Bold";
+    font-size: 16pt;
     border-radius: 3px;
 }
 
 .desktop_toggle_button {
     background-color: #91D064;
-    font: Bariol Bold 16;
+    font-family: "Bariol Bold";
```

Comment:
Good find with this monster

Reply:
Oh so do we change all of the instances to Bariol Bold? The other solution would be to find the Bariol font with embedded weights.

Reply:
I'll take a look at this whole bold issue - certainly the easiest thing right now is to just declare it as `Bariol Bold` but @wizofe you are right, we should look to do this properly.

Reply:
Given that Gtk is completely ignoring `font-weight`, I'll merge this as-is and make some alterations elsewhere to use `Bariol Bold` over `font-weight: bold`.